modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/toolbarbuttons/plugin.js

```diff
@@ -205,6 +205,71 @@
 					title: editor.lang.link.title,
 				});
 
+				/**
+				 * Spacing options are hardcoded to 1.0x, 1.5x, 2.0x, 3.0x, 4.0x and 5.0x,
+				 * so that an empty class, “mt-1 mb-1” class, up to “mt-5 mb-5” are applied respectively.
+				 * These classes define styles via the --spacer-* variables, associated with tokens “Spacer *”,
+				 * and so forth. Classic stylebook defines 11 spacing tokens (--spacer-0 to --spacer-10)
+				 */
+				/**
+				 * TODO:
+				 * 1. Find a way to apply a style to a selection :check:
+				 * 2. Try to host the state of the selected spacing in RichCombo title(Preserve title with the current selection on RichCombo)
+				 */
+				editor.ui.addRichCombo('LineSpacing', {
+					init() {
+						var instance = this;
```

Comment:
const?

Reply:
It's `var` all the way down here: CKEditor and it's plugins are written in ES5 only